src/platform/silabs/efr32/wifi/ethernetif.cpp

```diff
@@ -367,22 +404,37 @@ static err_t low_level_output(struct netif * netif, struct pbuf * p)
     /* Generate the packet */
     for (q = p, framelength = 0; q != NULL; q = q->next)
     {
+#if (SIWX_917 | EXP_BOARD)
+        wfx_rsi_pkt_add_data(packet, (uint8_t *) (q->payload), (uint16_t) q->len, framelength);
```

Comment:
Seems like the same prototype, Why the ifdef? Can't you just rename `rsipkt` to `packet`